backend/tests/services/rag/index/test_docling_indexer.py

```
@@ -0,0 +1,752 @@
+        assert config.heading_level_limit == 3
+        assert config.chunk_size == 1024
+        assert config.chunk_overlap == 50
+        assert config.separator is None
+        assert config.file_extension == ".pdf"
+
+    def test_docling_config_excel_json(self):
+        """Test DoclingPipelineConfig with Excel JSON format."""
+        config = DoclingPipelineConfig(
+            export_type="json",
+            file_extension=".xlsx",
+            separator="；\n",  # Fullwidth semicolon for Chinese documents
```

Ruff (0.14.14)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)